src/utils.test.js

```diff
@@ -0,0 +1,22 @@
+import { expect, it, describe } from "vitest";
+import { newDate, chomp /* mapChomp, Classifier  */ } from "./utils.js";
+
+describe("newDate", () => {
+  it("parses DD/MM/YYYY dates correctly", () => {
+    let d = newDate("10/12/1999");
+    expect(d).toEqual(new Date(1999, 12, 10));
+  });
+  ["10/12/19", "10/12/1999/2", "12/14/1999"].map((d) =>
+    it(`error on ${d}`, () => {
```

Comment:
This description doesn't help see immediately what is being tested or why

Reply:
tests cascade, it will say:
newDate error on ${string}